modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala

```diff
@@ -172,7 +172,7 @@ class packageSpec extends FlatSpec with NonImplicitAssertions {
             skip = false,
             List(
               List(fromTask(ActionType.Undefined,
-              Task(Result.Paused)))
+              Task(Result.Paused(new Result.PauseContext {} ))))
```

Comment:
Why are we using an anonymous class here? Could we create a class implementing `PauseContext`?

Reply:
Because it is only for tests.